includes/admin/class-give-settings.php

```diff
@@ -1414,10 +1414,10 @@ function give_api_callback() {
 	?>
 	<span class="cmb2-metabox-description api-description">
 		<?php echo sprintf(
-		/* translators: 1: https://givewp.com/documentation/give-api-reference/ 2: https://givewp.com/addons/zapier/ */
+		/* translators: 1: http://docs.givewp.com/api 2: http://docs.givewp.com/addon-zapier */
 			__( 'You can create API keys for individual users within their profile edit screen. API keys allow users to use the <a href="%1$s" target="_blank">Give REST API</a> to retrieve donation data in JSON or XML for external applications or devices, such as <a href="%2$s" target="_blank">Zapier</a>.', 'give' ),
-			esc_url( 'https://givewp.com/documentation/give-api-reference/' ),
-			esc_url( 'https://givewp.com/addons/zapier/' )
+			esc_url( 'http://docs.givewp.com/api' ),

```

Comment:
I'd love to write more docs about the API, so this one would be the root doc, and additional docs would be sub-pages and therefore need a hyphen, like `api-extending` That's why I kept it short.